src/js/api/config.js

```diff
@@ -87,6 +86,7 @@ const Config = function(options, persisted) {
         config.base = getScriptPath('jwplayer.js');
     }
     config.base = (config.base || loadFrom()).replace(/\/?$/, '/');
+    // eslint-disable-next-line
     __webpack_public_path__ = config.base;
```

Comment:
eslint didn't like use modifying a "global" here, but really we replace this at build time.